### PR TITLE
fix: add -oj -r to yq call

### PR DESCRIPTION
**Pull request by thesayyn**

### CODE REVIEW

File: bazel/container_structure_test.bzl [hunk 1]
```diff
@@ -29,7 +29,7 @@ readonly yq=$(rlocation {yq_path})
 
 # When the image points to a folder, we can read the index.json file inside
 if [[ -d "{image_path}" ]]; then
-  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
+  readonly DIGEST=$("$yq" eval -oj -r '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
```
Comment (alexeagle) [inline bazel/container_structure_test.bzl:32]: nit: maybe some comments about what the flags are for?

**Review by alexeagle — APPROVED** [review]

File: bazel/container_structure_test.bzl (hunk 1 above)
Comment (thesayyn) [reply bazel/container_structure_test.bzl:32]: sure. i'll do that.

Comment (thesayyn) [reply bazel/container_structure_test.bzl:34]: done

File: bazel/container_structure_test.bzl [hunk 2]
```diff
@@ -29,7 +29,9 @@ readonly yq=$(rlocation {yq_path})
 
 # When the image points to a folder, we can read the index.json file inside
 if [[ -d "{image_path}" ]]; then
-  readonly DIGEST=$("$yq" eval '.manifests[0].digest | sub(":"; "-")' "{image_path}/index.json")
+  # Never version of yq changed the default output to json and prints an annoying warning if -oj is not provided.
```
Comment (alexeagle) [inline bazel/container_structure_test.bzl:32]:
```suggestion
  # Newer versions of yq changed the default output to json and prints an annoying warning if -oj is not provided.
```

File: bazel/container_structure_test.bzl [hunk 3]
```diff
@@ -29,7 +29,9 @@ readonly yq=$(rlocation {yq_path})
...
+  # -oj simply means (-[o]output [j]son) and -r means raw output without the quotes.
```
Comment (alexeagle) [inline bazel/container_structure_test.bzl:33]: nit: if the flag has a long-form, that's better to use in a script like this so it's self-documenting

File: bazel/container_structure_test.bzl (hunk 2 above)
Comment (thesayyn) [reply bazel/container_structure_test.bzl]: done

File: bazel/container_structure_test.bzl (hunk 3 above)
Comment (thesayyn) [reply bazel/container_structure_test.bzl]: replaced -oj with long form.

**Comment by alexeagle** [comment]
Now that bazel-lib has the older yq version by default again, I think we should not land this as those new flags are not supported?

Or, it somehow needs to be conditional on what version of the yq toolchain is being used. (I could imagine bazel-lib could even add something to the YqInfo provider so the flags for "do json" are available along with the version being used)

**Pull request closed without merging**